src/ResourceManager/KeyVault/Commands.KeyVault/Models/DataServiceCredential.cs

```diff
@@ -94,8 +90,6 @@ private static string GetTenantId(IAzureContext context)
 
         private static Tuple<IAccessToken, string> GetTokenInternal(string tenantId, IAuthenticationFactory authFactory, IAzureContext context, string resourceIdEndpoint)
         {
-            if (string.IsNullOrWhiteSpace(tenantId))
```

Comment:
Why is this being removed?

Reply:
@markcowl This was removed because in the case that we don't get the tenant (with Account.Type being User or ServicePrincipal), I didn't want it to fail.

However, looks like we'd want to grab the tenant from the context in this case - does that sound reasonable? If so I'll go ahead and change it up that way.

Reply:
@tiffanyachen Correct, that would eb the correct thing to do here.